index.html

```diff
@@ -950,11 +956,253 @@ <h3>
+          </li>
+        </ul>
+        <p>
+          It is further critical that user agents design a permission
+          experience that accounts for the lack of these mitigations, e.g. the
+          exchange of personal information from government credentials without
+          any verifier authentication scheme. It is recommended to apply a very
+          high level of friction for these types of high risk exchanges.
+        </p>
+        <h4>
+          Non-government-issued credentials
+        </h4>
+        <p>
+          Non-government-issued credentials include all other digital
+          documents, certificates and attestations that are not
+          government-issued and don't represent government-issued documents.
+          This could include proof of employment, education credentials or
+          cinema tickets. Notably, their exchange is likely less restricted
+          through laws and regulations. While these documents often don't
+          exhibit the same risks as government-issued credentials, they could
+          also contain identifiable or sensitive information.
+        </p>
+        <h5>
+          Risk of theft and leakage of non-government credentials
+        </h5>
+        <p>
+          The impact and viability of credential theft and leakage for
+          non-government credentials is largely based on the content of each
+          individual credential type. In general, it could lead to loss of
+          control and exposure of sensitive private information, as well as
+          impersonation and data theft, which can increase the likelihood of
+          further attacks on the affected individual.
+        </p>
+        <h5>
+          Risk of proliferation of requests for government credentials
+        </h5>
+        <p>
+          The flexibility and lack of regulation of non-government credentials
+          carries potential for abuse for the purpose of cross-site tracking
+          and linking identities, through long-lived identifiers such as email
+          address or phone number. Verifiers participating in a tracking scheme
+          based on digital credentials could create user incentives to accept
+          sharing identifier credentials across many sites ("loyalty cards" for
+          the Web), without fully understanding the implications on their
+          privacy.
+        </p>
+        <p>
+          Even users unwilling to share their information in such a scheme
+          could be affected by prompt fatigue and potentially risk exclusion
+          from using these services.
+        </p>
+        <h5>
+          Mitigating unnecessary requests for non-government credentials
+        </h5>
+        <p>
+          For non-government-issued credentials, it is especially critical that
+          the user agent understands the requested credential format and its
+          privacy attributes, and builds a risk framework that informs the the
```

Comment:
Unfortunately I don't think this is scalable in practice. We're seeing a proliferation of non-government-issued credentials for various purposes and I can't really imagine having the browser understand most of them. Maybe we can say it's critical for browsers to understand the privacy properties of the most popular forms of credentials in practice to help support as many users as possible in deciding how to engage?

Reply:
Yeah, reading this again I agree that it's too strongly worded. I can update this to say that the user agent should understand at least the most popular requests and adjust the communicated risk level to users when it does not understand a request.